migration_scripts/bulk_load/src/hybrid_tx_benchmark.clj

```diff
@@ -246,28 +294,29 @@
            kind           ;; transaction kind
            sign recv      ;; addresses
            input changed  ;; objects
-           ]}]
+           ]}
+   lo hi]
   (let [sub-queries
         (cond-> []
           (and pkg mod fun)
-          (conj (select-fun params))
+          (conj (select-fun params lo hi))
 
           (and pkg mod (not fun))
-          (conj (select-mod params))
+          (conj (select-mod params lo hi))
 
           (and pkg (not mod) (not fun))
-          (conj (select-pkg params))
+          (conj (select-pkg params lo hi))
 
           (and kind (not sign))
-          (conj (select-kind params))
+          (conj (select-kind params lo hi))
 
           ;; No explicit case for `sign` because sender filters are incorporated
           ;; into each select. Assertion below guarantees at least one select
           ;; used.
 
-          recv    (conj (select-recipient params))
-          input   (conj (select-input params))
-          changed (conj (select-changed params)))
+          recv    (conj (select-recipient params lo hi))
+          input   (conj (select-input params lo hi))
+          changed (conj (select-changed params lo hi)))
```

Comment:
the compound-norm implementation is missing a line like ids (conj (select-ids params lo hi))